proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
Could you try the same `proxy.json` against a non-Docker version of the proxy? I believe this particular issue is being caused by the invalid `realm-public-key`, so, it should also happen with the non-Docker version.

In theory, the documentation linked from the readme file should still work for this proxy, there's nothing really specific to the Docker version.